src/components/BulkOperationDetailsPage/styles.scss

```diff
@@ -0,0 +1,37 @@
+.csv-preview-modal {
+
+    $cell-widths: 220px;
```

Comment:
nit: Conventionally, this variable should be at the top... Outside `.csv-preview-modal`